src/index.js

```diff
@@ -11,10 +11,16 @@ const defaultOptions = {
 const mergeImages = (sources = [], options = {}) => new Promise(resolve => {
 	options = Object.assign({}, defaultOptions, options);
 
+	if (options.Canvas && !options.Image && !options.Canvas.Image) {
+		// See https://github.com/Automattic/node-canvas/blob/master/CHANGELOG.md#200
+		throw new Error('"options.Image" is needed. You probably want `import { Image } from "canvas"`');
+	}
+
 	// Setup browser/Node.js specific variables
 	const canvas = options.Canvas ? new options.Canvas() : window.document.createElement('canvas');
-	const Image = options.Canvas ? options.Canvas.Image : window.Image;
-	if (options.Canvas) {
+	const Image = options.Image || (options.Canvas ? options.Canvas.Image : window.Image);
+	if (options.Canvas && options.Canvas.version && options.Canvas.version.indexOf('1.') === 0) {
```

Comment:
I think we can remove this. No need to support `canvas@1` anymore.

Reply:
Cool will drop the v1 support :+1:

Reply:
done